docs/working_with_page.md

```diff
@@ -0,0 +1,226 @@
+    Each block is unique in its function and enables you to personalize the content appearing on the Page.
+    
+    ![Dragging a block onto a zone](img/dragging_lp_block.png "Dragging a block onto a zone")
+    
+    Add a block by dragging it from the menu to an empty place on a zone.
+    Do not worry about placing blocks in the proper place from the start.
+    You can reorder them at any time by dragging and dropping them in the desired location.
+    
+    eZ Enterprise comes with a set of ready-to-use Page blocks,
+    but [developers can add their own](https://doc.ezplatform.com/en/latest/guide/extending_page/#creating-page-blocks).
+    
+    ![Elements menu](img/elements_menu.png "Elements menu")
+    
+    The blocks provided with a clean installation of eZ Enterprise are:
+    
+    |Block|Description|
+    |-----|-----------|
+    |Form|Embeds a Form Content item that you select from the Content Structure.|
+    |Code|Enables you to place text, links, images, etc. on your Page using custom HTML.|
+    |Content List|Displays Content items of a chosen Content Type (or Types) that are contained in a selected folder.|
+    |Banner|Displays an image Content item with a URL attached to it.|
+    |Collection|Displays a set of Content items you select manually from the Content structure.|
+    |Embed|Embeds a Content item of any Content Type on the Page.|
+    |Gallery|Displays all images contained in a selected folder.|
+    |Video|Embeds a video into the Page with standard playback controls.|
+    |RSS|Loads and displays news from RSS feeds (channels).|
+    |Content Scheduler|Displays Content items at a pre-define time.|
+    |Text|Enables you to add to the Page a Rich Text block.|
+    
+    ## Block settings
+    
+    ### Blocks of content
+    
+    Each kind of block has its special properties. 
+    You can access them by placing the cursor on the added block and clicking the cog button.
+    Use the same menu to remove the block from the Page by clicking the trash button.
+    
+    ![Elements settings](img/page_builder_elementsettings.png)
+    
+    Settings available for blocks are divided into three tabs — **Basic**, **Design**, and **Scheduler**.
+    The settings available in the **Basic** tab are tailored to the block's content.
+    The remaining tabs contain parameters common to all blocks provided with eZ Enterprise. 
+    Use them to modify the layout and visibility of a block.
+    Always ensure that you set all the required settings of a block of content to successfully publish the Page.
+        
+    In the **Design** tab, perform the following actions:
+    
+    - From the **View** list, select the block layout.
+    - *Optional:* In the **Class** box, define the class of the block
+    - *Optional:* In the **Style** box, add additional style formatting.
+    
+    In the **Scheduler** tab, define the visibility time of the block by performing the following actions:
```

Comment:
This is already covered in publishing/advanced_publishing_options/, so maybe just a link?